app/controllers/api/v1/organizations_controller.rb

```diff
@@ -16,6 +16,11 @@ class Api::V1::OrganizationsController < Api::ApiController
                     :title,
                     :introduction].freeze
 
+  CONTENT_FIELDS = [:description,
```

Comment:
The shared concern isn't organization-specific, which is why I'm passing the `CONTENT_FIELDS` array in via argument to the shared method in the operation. That way, if we do ever migrate Projects to use service objects too, they can use the same concern.